submissions/tests/test_team_api.py

```diff
@@ -194,24 +194,58 @@ def test_create_submission_for_team_invalid_attempt(self):
     def test_create_submission_for_team_existing_individual_submission(self):
         """
         Test for calling create_submission_for_team when a user somehow already has
-        an existing active submission for the item.
-
-        For normal Submissions, if a submission exists and we create another one, the second
-        will increment the first's attempt_number. However, for team submissions, we currently
-        pass the attempt_number from team_api.create_submission to api.create_submission, so
-        all created individual submissions will have the same attempt_number
+        an existing active submission for the item in the same course.
+        This can happen is a user was on a team that submitted and then reassigned to a team
+        that didn't have a submission and is submitting.
+        Expected outcome: submission is created because the the previous submission was not team based
```

Comment:
Grammar/typo fixes. Also, an item is by definition in the same course so you can remove that:
```suggestion
        an existing active submission for the item.
        This can happen if a user was on a team that submitted and then is reassigned to a team
        that didn't have a submission and is submitting.
        Expected outcome: submission is created because the the previous submission was not team based
```